force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -62,14 +68,28 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Boolean> checkStringContent(String input) {
         // Initialize a list to store the results
-        
+        List<Boolean> result = new List<Boolean>();
+
+        if(input.startsWith('Sales')){
+            result.add(true);
+        } else {
+            result.add(false);
+        }
```

Comment:
```suggestion
Boolean startsWithSales = input.startsWith('Sales')
results.add(startsWithSales );
```
This might be a little easier to read and less wordy